Content.Server/GameObjects/Components/Tag/TagComponent.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System.Collections.Generic;
```

Comment:
plz nullable

Reply:
For what sir
Should I make the methods accept nullable strings?